docs/source/api/index.rst

```diff
@@ -153,29 +153,42 @@ Template
 Basic
 ~~~~~
 
```

Comment:
Would be interesting to add a "Class-based interface" here

Reply:
Mmh not sure I understand?